src/main/java/edu/cornell/kfs/fp/batch/service/CuProcurementCardLoadTransactionsService.java

```diff
@@ -0,0 +1,7 @@
+package edu.cornell.kfs.fp.batch.service;
+
+import org.kuali.kfs.fp.batch.service.ProcurementCardLoadTransactionsService;
+
+public interface CuProcurementCardLoadTransactionsService extends ProcurementCardLoadTransactionsService {
```

Comment:
I am not sure I understand why this CuProcurementCardLoadTransactionsService is needed and not a CorporateBilledCorporatePaidLoadTransactionService instead. Then the loadProcurementCardFile could also be renamed to loadCorporateBilledCorporatePaidFile.

Reply:
This actually isn't used in CBCP processing, it is needed for unit testing in ProcurementCardCreateDocumentServiceImplTest and ProcurementCardLoadFlatTransactionsServiceImplTest.  These aren't micro tests, and load spring.  
The original tests got ProcurementCardLoadTransactionsService from Spring and cast it to ProcurementCardLoadFlatTransactionsServiceImpl so the test could call loadProcurementCardFile.  That cast to ProcurementCardLoadFlatTransactionsServiceImpl was failing after my changes.  I suspect it had something to do with there being multiple services that implement ProcurementCardLoadTransactionsService now.

Reply:
That is strange, but it seems like a workaround rather than a fix. You've been changing this implementation since that unit test failure, would the unit tests still fail now if you removed this interface?

Reply:
I can try